### ISSUE TITLE
Question about mean iou and loss ?

### ISSUE THREAD

**Issue by ThorPham** [gujingxiao/Lane-Segmentation-Solution-For-Baidu-PaddlePaddle-Autonomous-Driving-Competition#4]
when i train continuous from pretrain model i get mean IOU is 1 and loss = 0 . but when i evaluation get very bad result . What's wrong ?
loaded model from: ./model_weights/paddle_deeplabv3p_8_end_060223
Start Training Epoch: 1
Iter - 100: train loss: 0.003, mean iou: 0.855, time cost: 190.493 s
Iter - 200: train loss: 0.000, mean iou: 0.995, time cost: 164.124 s
Iter - 300: train loss: 0.000, mean iou: 1.000, time cost: 163.251 s
Iter - 400: train loss: 0.000, mean iou: 1.000, time cost: 163.802 s
Iter - 500: train loss: 0.000, mean iou: 1.000, time cost: 167.394 s
Iter - 600: train loss: 0.000, mean iou: 1.000, time cost: 167.161 s
Iter - 700: train loss: 0.000, mean iou: 1.000, time cost: 167.958 s
Iter - 800: train loss: 0.000, mean iou: 1.000, time cost: 165.983 s
Iter - 900: train loss: 0.000, mean iou: 1.000, time cost: 164.810 s
Iter - 1000: train loss: 0.000, mean iou: 1.000, time cost: 166.612 s

**Comment by gujingxiao** [comment]
Hi ThorPham, if possible, could you please briefly describe the data preparation procedure when you do training? I need more information.

**Comment by ThorPham** [comment]
@gujingxiao  I make file csv same folder Data_list your code. And run file train.py . I don't modify any thing  your code .

**Comment by ThorPham** [comment]
@gujingxiao I have a question about data_feeder : 
https://github.com/gujingxiao/Lane-Segmentation-Solution-For-BaiduAI-Autonomous-Driving-Competition/blob/0c9ec7e2cbb658849cb0d29e05252e237d77663a/utils/data_feeder.py#L42
When i count value pixel in image label using function  unique= np.unique(ori_mask) take output : array([ 0, 29, 89, 138, 154, 157], dtype=uint8) . Output not match with color_mark in function encode_labels .
https://github.com/gujingxiao/Lane-Segmentation-Solution-For-BaiduAI-Autonomous-Driving-Competition/blob/0c9ec7e2cbb658849cb0d29e05252e237d77663a/utils/process_labels.py#L9
 I don't know why ?. I think value pixel must same color_mark ?

**Comment by gujingxiao** [comment]
The competition provide two formats for the mask. For my case, use the gray one.

You may find some useful information via this link: https://aistudio.baidu.com/aistudio/competition/detail/5. Some developers use my pretrained model and this project also, but they got normal results. Maybe the competition and this project use Chinese descriptions which cause you some misunderstanding.

Feel free to post bugs & questions. This project may indeed have lots of bugs, Haha.

**Comment by ThorPham** [comment]
Thank you so much. I will check again

**Comment by kong-ku** [comment]
I got the same problem.Do you solve it？

**Comment by ThorPham** [comment]
I have fixed it . You must read image which mode RGB and make mask from it not gray

**Comment by gujingxiao** [comment]
Awesome, you got it!

**Comment by STRIVESS** [comment]
> 
> 
> I have fixed it . You must read image which mode RGB and make mask from it not gray

Hello,ThorPham,nice to meet you!  I am greatly cheerful to recognize you in your GitHub.Whether you ever have finished the training in your comuputer or not? My computer have no GPU and I don't support the training,so can you are convinient to share your pretrain model,including the other two unet models? Thank you very much
